cmd/bisync/rc.go

```diff
@@ -0,0 +1,92 @@
+package bisync
+
+import (
+	"context"
+	"log"
+
+	"github.com/pkg/errors"
+	"github.com/rclone/rclone/cmd/bisync/bilib"
+	"github.com/rclone/rclone/fs"
+	"github.com/rclone/rclone/fs/rc"
+)
+
+func init() {
+	rc.Add(rc.Call{
+		Path:         "sync/bisync",
+		AuthRequired: true,
+		Fn:           rcBisync,
+		Title:        shortHelp,
+		Help:         rcHelp,
+	})
+}
+
+func rcBisync(ctx context.Context, in rc.Params) (out rc.Params, err error) {
+	opt := &Options{}
+	octx, ci := fs.AddConfig(ctx)
+
+	if dryRun, err := in.GetBool("dryRun"); err == nil {
+		ci.DryRun = dryRun
+	} else if rc.NotErrParamNotFound(err) {
+		return nil, err
+	}
+
+	if maxDelete, err := in.GetInt64("maxDelete"); err == nil {
+		if maxDelete < 0 || maxDelete > 100 {
+			// TODO Cannot return rc.ErrParamInvalid as its field is private
```

Comment:
@ncw 
I would like to return a `ErrParamInvalid` error here but it won't let me set a required field.
May we change its definition and make the field public?